### ISSUE TITLE
[BETA-4] IC2 jetpack can be crafted with Power Armor.

### ISSUE THREAD

**Issue by eyeonus** [MachineMuse/MachineMusePowersuits#807]
I'm pretty sure IC2 is actually the cause of this weirdness, but I honestly don't know which mod needs the fixin' in it, so....

Verified with the following versions:
industrialcraft-2-2.6.234-ex110
ModularPowersuits-1.10.2-1.2.120 (beta-4)

I didn't find this out myself, I saw it on a Let's Play:
https://www.youtube.com/watch?v=jqEPGfp7hsE&t=345s

**Comment by lehjr** [comment]
I've seen that. It's been around for awhile now. It's an IC2 thing. I'm not sure how to fix it other than on their end. To me it looks like they're just blindly adding any chestplate to the recipe. And it's not an open source mod, so you can't submit a PR. The only options are to submit a bug report and see what happens there or look at the deobfuscated code and figure out a workaround.

**Comment by eyeonus** [comment]
I figured it was something like that, I just thought you should know if you didn't already.

**Comment by lehjr** [comment]
Yeah, I haven't really looked in to how to fix it yet, but I'll get to it Soon(TM).

**Comment by eyeonus** [comment]
No worries©, mate™. :)
